tasks/setup.py

```diff
@@ -204,6 +205,21 @@ def enable_pre_commit(ctx) -> SetupResult:
     return SetupResult("Enable pre-commit", status, message)
 
 
+def setup_vscode(ctx) -> SetupResult:
+    print(color_message("Setting up VS Code...", Color.BLUE))
+
+    status = Status.OK
+    message = ""
+
+    if yes_no_question("Do you want to set up VS Code?", default=True):
```

Comment:
I would add a parameter (which defaults to false) to the setup task instead of asking it interactively. If we use prompts we won't be able to use the setup task in scripts later if needed. What do you think?

Reply:
This remark is valid for `yes_no_question` calls in vscode.py as well I guess

Reply:
`yes` and `no` haha I do not expect to configure vscode in a CI for instance, so one could argue the VS code part can be interactive. I would do the same thing in vscode.py as you suggest though because I don't like being asked things interactively 😅 but that's personal preferences

Reply:
True, I removed the assert. I will also catch other exceptions 👍